### PR TITLE
WIP: Linux bridge vlan filter

### PR DESCRIPTION
**Pull request by maiqueb**

### CODE REVIEW

**Comment by maiqueb** [comment]
Passing on build against networkmanager-1.18, somehow failing against master.
https://travis-ci.org/nmstate/nmstate/jobs/560003493

File: libnmstate/schemas/operational-state.yaml [hunk 1]
```diff
@@ -457,3 +467,14 @@ definitions:
         type: array
         items:
           type: string
+  vlan-filter:
+    type: object
+    properties:
+      vlan-range-min:
+        type: integer
+        minimum: 0
```
Comment (qinqon) [inline libnmstate/schemas/operational-state.yaml]: I think this is 1 as stated at my previous comment

File: libnmstate/schemas/operational-state.yaml [hunk 2]
```diff
@@ -457,3 +467,14 @@ definitions:
...
+        maximum: 4095
+      vlan-range-max:
+        type: integer
+        minimum: 0
```
Comment (qinqon) [inline libnmstate/schemas/operational-state.yaml]: ditto

File: libnmstate/schemas/operational-state.yaml (hunk 1 above)
Comment (maiqueb) [reply libnmstate/schemas/operational-state.yaml]: Done.

File: libnmstate/schemas/operational-state.yaml (hunk 2 above)
Comment (maiqueb) [reply libnmstate/schemas/operational-state.yaml]: Done.

File: libnmstate/schemas/operational-state.yaml (hunk 1 above)
Comment (maiqueb) [reply libnmstate/schemas/operational-state.yaml]: Fixed in https://github.com/nmstate/nmstate/pull/440/

File: libnmstate/nm/bridge.py [hunk 3]
```diff
@@ -57,6 +57,12 @@ def _set_bridge_properties(bridge_setting, options):
             bridge_setting.props.multicast_snooping = val
         elif key == LB.STP_SUBTREE:
             _set_bridge_stp_properties(bridge_setting, val)
+        elif key == LB.VLAN_FILTERING:
+            bridge_setting.props.vlan_filtering = val
+            bridge_setting.props.vlan_default_pvid = 0
```
Comment (qinqon) [inline libnmstate/nm/bridge.py]: default pvid is 1 by default :-)

**Comment by maiqueb** [comment]
Closing the PR, since the accepted API changed drastically from what's being developed here. 

Easier to just open a new one in the future.

**Pull request closed without merging**